exercise_04.py

```diff
@@ -0,0 +1,23 @@
+def found_pi(number):
+    formula = 0
+
+    for i in range(1, number + 1):
+        formula += (-1) ** ( i + 1) / (2 * i - 1)
+        pi = 4 * formula
+
+    return f"{pi:.4f}"
+
+
+def pi_values_table(n):
+    print(f"{'i':<25}m(i)\n")
+    for i in range(1, n + 1, 100):
+        to_print = f"{i:<25}{found_pi(i)}"
+        print(to_print)
+
+
+def main():
+    return pi_values_table(901)
```

Comment:
`pi_values_table` returns `None`, so why the hell in the world you return `None` from `main`. Just call the function, and pay attention to what you are returning. Otherwise, your code looks gooky!